protocol/avdl/chat1/local.avdl

```diff
@@ -292,6 +292,11 @@ protocol local {
     // This is the only difference between V2 and V3.
     @mpackkey("em") @jsonkey("em")
     union { null, MsgEphemeralMetadata } ephemeralMetadata;
+
+    // Whether or not the message was signed using pairwise MACs. Only valid
+    // for V4 where pairwise MACs are used for exploding messages.
+    @mpackkey("pm") @jsonkey("pm")
+    union { null, boolean } hasPairwiseMacs;
```

Comment:
Why does this need to be here? We can't just get this information from somewhere else?